src/foam/core/crunch/lite/CapableDAO.js

```diff
@@ -42,6 +43,57 @@ foam.CLASS({
   ],
 
   methods: [
+    {
+      name: 'find_',
+      javaCode: `
+        Capable capable = (Capable) getDelegate().find_(x, id);
+        if ( capable == null ) {
+          return null;
+        }
+
+        capable = populatePayloads(x, capable);
+
+        return (FObject) capable;
+      `
+    },
+    {
+      name: 'select_',
+      javaCode: `
+        if (sink != null) {
+          ProxySink refinedSink = new ProxySink(x, sink) {
+            @Override
+            public void put(Object obj, foam.lang.Detachable sub) {
+              Capable capable = (Capable) ((FObject) obj).fclone();
```

Comment:
```suggestion
              Capable capable = (Capable) obj;
```

Reply:
This is needed for super.put(....), in case payloads are empty